recipes/traffix/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "traffix" %}
+{% set version = "0.1.11a0" %}
```

Comment:
Does this package have any regular releases (i.e. non-alpha/beta)? We prefer to package non-dev releases on the main channel.

Reply:
Actually it's already passed alpha and beta stage so I changed it into regular package.